🌐 Update Danish Translations

Christian-Schou opened the pull request:
This update synchronizes the Danish translations with the latest version of Ghost, ensuring that all text strings are accurately translated and up-to-date.

Christian-Schou committed:
Update comments DA

Christian-Schou committed:
Update ghost DA translations

Christian-Schou committed:
Update newsletter DA translations

Christian-Schou committed:
Update portal DA translations

Christian-Schou committed:
Merge branch 'main' into main-chs-update-danish-translations

Christian-Schou committed:
Merge branch 'main' into main-chs-update-danish-translations

cathysarisky reviewed (changes requested):
Thanks for the translations!  

I used AI tooling to check over these.

One minor possible error noted, but everything else looks good.  (If the AI is wrong -- which happens a lot -- please say so and I'll get it approved.)

Christian-Schou commented on ghost/i18n/locales/da/portal.json:
```diff
@@ -179,7 +179,7 @@
     "Too many different sign-in attempts, try again in {{number}} hours": "For mange forskellige log-in forsøg, prøv igen om {{number}} timer.",
     "Too many different sign-in attempts, try again in {{number}} minutes": "For mange forskellige log-in forsøg, prøv igen om {{number}} minutter.",
-    "Too many sign-up attempts, try again later": "",
+    "Too many sign-up attempts, try again later": "For mange log-in forsøg, prøv venligst igen senere",
```
Great catch, I will fix that 💯

Christian-Schou committed:
🐛 Fixed translation

Changed the sign-up translation and found a wrong translation for the sign-in part.

Christian-Schou committed:
Merge branch 'main' into main-chs-update-danish-translations

ErisDS committed:
Merge branch 'main' into main-chs-update-danish-translations
